common/c_cpp/src/c/wombat/wtable.h

```diff
@@ -31,6 +31,8 @@ extern "C" {
 
 typedef void* wtable_t;
 
+/* Indicate the success of wTable functions. */
+#define WTABLE_INSERT_SUCCESS   1
```

Comment:
This doesn't look like it's used anywhere, although it may be useful, can it be taken out of this commit to keep it clean?